.vscode/settings.json

```diff
@@ -7,7 +7,7 @@
     "**/.git": true,
     "**/.DS_Store": true,
     "**/node_modules/**": true,
-    "**/build/**": true,
+    "**/build/**": false,
```

Comment:
why?

Reply:
Not being able to see into the build directory caused me to miss all the changes that had happened with `ng-packagr` for their release for v13